src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java

```diff
@@ -0,0 +1,13 @@
+package com.carleton.CapstoneSystem.models;
+
+import org.junit.Test;
+
+import javax.persistence.Entity;
+import javax.persistence.Inheritance;
+
+public class ProfessorTest {
+    @Test
```

Comment:
we should be testing all the getters and setters too technically. That's why the code coverage is complaining.

Reply:
ok I will make the right tests for those

Reply:
same comment as tosin